File "./output/apply_with.v", line 3, characters 11-26:
The command has indeed failed with message:
No such bound variable d (possible names are: a, b and c).
